github/ci/prow/files/jobs/hyperconverged-cluster-operator/hyperconverged-cluster-operator-presubmits-2.3.yaml

```diff
@@ -0,0 +1,64 @@
+periodics: null
+postsubmits: {}
+presubmits:
+  kubevirt/hyperconverged-cluster-operator:
+  - always_run: true
+    branches:
+    - release-2.3
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 7h0m0s
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror: "true"
+      preset-shared-images: "true"
+    max_concurrency: 6
+    name: pull-hyperconverged-cluster-operator-e2e-k8s-1.15.1
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - export TARGET=k8s-1.15.1 && automation/test.sh
+        image: docker.io/kubevirtci/kubevirt-infra-bootstrap:v20191223-f0a2baa
+        name: ""
+        resources:
+          requests:
+            memory: 29Gi
+        securityContext:
+          privileged: true
+      nodeSelector:
+        type: bare-metal-external
+  - always_run: true
+    branches:
+    - release-2.3
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 7h0m0s
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror: "true"
+      preset-kubevirtci-quay-credential: "true"
+      preset-shared-images: "true"
+    max_concurrency: 6
+    name: pull-hyperconverged-cluster-operator-e2e-ocp-4.3
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/bash
+        - -c
+        - cat $QUAY_PASSWORD | docker login --username $(cat $QUAY_USER) --password-stdin=true
+          quay.io && export TARGET=ocp-4.3 && automation/test.sh
```

Comment:
@gbenhaim, I just want to understand - where is the `automation/test.sh` code located?

Reply:
https://github.com/kubevirt/hyperconverged-cluster-operator/tree/master/automation